src/jit.rs

```diff
@@ -1446,6 +1496,55 @@ impl<'a, C: ContextObject> JitCompiler<'a, C> {
 
             self.emit_ins(X86Instruction::return_near());
         }
+
```

Comment:
These different code paths are also not needed. The previous anchors will do. Just change the called function pointer depending on `len` and the access type.

Reply:
done